projects/packages/publicize/src/class-keyring-service-helper.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Utilities to interact with a Keyring instance.
```

Comment:
Should we opt in to move the existing `Jetpack_Keyring_Service_Helper` from the Jetpack plugin to the package?

If we're not ready to require the Publicize package in Jetpack, maybe we should add a mention here and in the original file to make sure that any changes made to one package are reflected to the other one?

Reply:
@jeherve I believe we might not need the `Jetpack_Keyring_Service_Helper` at all—I need to confirm but I think it's a remnant from when the sharing settings lived in wp-admin. This would also solve some of the other concerns you mentioned.

I'll do some research and see if we're able to just remove it from the package altogether.

Reply:
@ebinnion FYI, we might not need to do anything with the Keyring helper. I'm looking into it.

Reply:
Sounds good, I'm happy to take over from here and I appreciate you getting the ball rolling here 🙏🏼

> But, there are some bits that call in from the class-publicize.php file that make me think that the file probably needs to exist in its current state.

I believe these functions in `class-publicize.php` might be dead code, which would remove the dependency on the Keyring helper class entirely. I'm testing to see if it gets used anywhere.